### DISCUSSION TITLE
Hey just wanted to share a project I'm making using hex grid

### DISCUSSION THREAD

**Discussion by theslantedroom** [Hellenic/react-hexgrid#114] [General]
It is a hex grid turn based boxing game in very early stages of development. 
https://hbc-boxing.meteorapp.com
I have some storybooks too that show examples of moving on a hex grid if that interests anyone.  Could make a pr with an example or two

**Comment by Dissolutio** [comment]
That game is awesome, well done!

**Reply by theslantedroom** [reply]
Hey man. Take a look were I'm at now with the game half a year later. Thanks again for react-hex grid

www.hbcboxing.online

Lot of work yet to do but getting closer.
